cmd/kubeadm/app/constants/constants.go

```diff
@@ -410,7 +417,10 @@ var (
 		13: "3.2.24",
 		14: "3.3.10",
 		15: "3.3.10",
-		16: "3.3.10",
+		16: "3.3.17",
+		17: "3.4.3",
```

Comment:
why there are two 3.3.10 and 3.4.3?

Reply:
Also why adding 3.4.3 here?

Reply:
It is the etcd version that kubernetes supported. kubernetes 17 supports 3.4.3. I am little hesitate to make changes here. Since the arktos is kubernetes 14.x, I shall make changes here.

Reply:
Talked to Niaoying and will used 3.4.4 as arktos etcd version from now.